Sources/TuistGraph/Models/FileElement.swift

```diff
@@ -23,3 +23,42 @@ public enum FileElement: Equatable, Hashable {
         }
     }
 }
+
+// MARK: - Codable
+
+extension FileElement {
+    private enum Kind: String, Codable {
+        case file
+        case folderReference
+    }
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
I believe you can declare `enum CodingKeys` private as well (might be worth adding it to other parts of the PR as well if that's true)